third_party/amd/lib/TritonAMDGPUToLLVM/TargetInfo.h

```diff
@@ -21,11 +21,12 @@ class TargetInfo : public mlir::triton::TargetInfoBase {
   Value ballot(RewriterBase &rewriter, Location loc, Type type,
                Value cmp) const override;
 
-  void storeShared(RewriterBase &rewriter, Location loc, Value ptr, Value val,
-                   Value pred) const override;
-  Value loadShared(RewriterBase &rewriter, Location loc,
-                   const TypeConverter *converter, Value ptr, Type elemTy,
-                   Value pred) const override;
+  void storeDShared(RewriterBase &rewriter, Location loc, Value ptr,
```

Comment:
my understanding was that this was only in Nvidia backend because it is specific to Nvidia backend and it doesn't exist on AMD side.

Reply:
Please let me know if the comment in TargetInfoBase.h doesn't explain it.

Reply:
Talked to Thomas offline, he said to go ahead with this.